sound/soc/sof/intel/hda-loader.c

```diff
@@ -184,8 +184,7 @@ static int cl_dsp_init(struct snd_sof_dev *sdev, int stream_tag)
 			__func__);
 
 err:
-	flags = SOF_DBG_DUMP_REGS | SOF_DBG_DUMP_PCI | SOF_DBG_DUMP_MBOX |
```

Comment:
typo in commit message " downloading the firmware to DSP"

Reply:
Thanks!